Dockerfile

```diff
@@ -223,7 +237,7 @@ RUN \
   wget -O lame.tar.gz "$MP3LAME_URL" && \
   echo "$MP3LAME_SHA256  lame.tar.gz" | sha256sum --status -c - && \
   tar xf lame.tar.gz && \
-  cd lame-* && ./configure --disable-shared --enable-static --enable-nasm --disable-gtktest --disable-cpml --disable-frontend && \
+  cd lame-* && ./configure --disable-shared --enable-static --enable-nasm --disable-gtktest --disable-cpml --disable-frontend --disable-rpath && \
```

Comment:
Will rpath stuff affect static PIE builds? thinking rpath is usually a dynamic lib runtime linking thingy or?